src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -26,6 +27,8 @@ export class FileExplorerComponent implements OnInit {
   @Output('done') done: EventEmitter<any> = new EventEmitter();
 
   files: TreeNode[];
+  item: Item;
```

Comment:
At this point I think we need comments for each field in the component to describe what state it represents. `files` seems pretty clear, it's the model that this component is drawing onscreen. `selectedFiles` is also pretty clear, it's the state of the current selection. But things like `item` and `downloadItems` and `archivos` are less clear to me.

Reply:
Sorry, you're totally right. I'll forget to change those names.